bin/tessel-2.js

```diff
@@ -19,9 +19,12 @@ const CLI_ENTRYPOINT = 'cli.entrypoint';
 
 // Check for updates
 const pkg = require('../package.json');
-updateNotifier({
-  pkg
-}).notify();
+
+try {
+  updateNotifier({ pkg }).notify();
+} catch (err) {
+  CrashReporter.submit(err.stack, { silent: true });
```

Comment:
I thought it was a good idea to keep collecting the crash data, that way we can see how many users are impacted by this, and also help out with additional information that might be required for yeoman/update-notifier#76...